.github/workflows/deploy-lambdas.yml

```diff
@@ -0,0 +1,102 @@
+name: Deploy lambdas
+
+on:
+  workflow_call:
+    inputs:
+      ENV:
+        required: true
+        type: string
+      SERVICE:
+        required: true
+        type: string
+    secrets:
+      SLACK_NOTIFICATION_CI_RUNS:
+        required: true
+      ACCESS_TOKEN:
+        required: true
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+      AWS_DEFAULT_REGION:
+        required: true
+
+jobs:
+  deploy:
+    name: Deploy Staging Lambdas
+    runs-on: ubuntu-latest
+    container:
+      image: timbru31/node-alpine-git
+    env:
+      ENV: ${{ inputs.ENV }}
+      SERVICE: ${{ inputs.SERVICE }}
+      ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_DEFAULT_REGION: ${{secrets.AWS_DEFAULT_REGION}}
+      SLACK_NOTIFICATION_CI_RUNS: ${{ secrets.SLACK_NOTIFICATION_CI_RUNS }}
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+      - name: Set NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: .nvmrc
+          cache: npm
+      - name: 'NPM: Add Config and Authorization'
+        run: |
+          rm -f .npmrc
```

Comment:
The `.npmrc` file might not exist which would cause rm to exit with a non-zero exit code. Also redirecting the echo command output below, using `>`, will surely override the contents of `.npmrc`.

Reply:
AFAIK the -f flag means that if it doesn't exist it won't error. Also as I remember I had found that `>` wasn't overriding the original file... quite likely I made some sort of mistake though

Reply:
You're right about the `-f` flag. Although it can also be used for ignoring confirmation prompts and similar. The redirection should work on the other hand. One possibility is that the original `.npmrc` file is protected and as such cannot be overriden, thus must be forcefully removed and recreated. Though I don't think that's likely.